config/dynamic-scaling-configuration.yaml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
When we've the final config it might be good to add comments describing the structure of the config. 
Just like https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/8e1c36ecb801a26666aad7d1ec90f06411d97fbf/config/kafka-instance-types-configuration.yaml#L1

Reply:
:+1: